### PR TITLE
Update Replicate Extension

### PR DESCRIPTION
**Pull request by cbh123**
## Description

Updates @KevinBatdorf's Replicate extension. Now users can:
- Run models on Replicate
- View prediction data
- Explore models / open replicate dashboard 

## Screencast

https://www.loom.com/share/7afb734f7a814829af8655ecb5470108

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by zeke** [comment]
Replicate employee here. 👋🏼 

Took this for a spin and it works like a charm. Looks good to me!

cc @pernielsentikaer @mathieudutour

File: extensions/replicate/package.json [hunk 1]
```diff
@@ -16,10 +17,10 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "replicate",
-      "title": "View Predictions",
-      "subtitle": "Replicate",
-      "description": "View your Replicate predictions",
+      "name": "index",
```
Comment (pernielsentikaer) [inline extensions/replicate/package.json:20]: Could we NOT change this if it's not really important? When this is changed people loses their hotkey/aliases/ranking for the command

Comment (KevinBatdorf) [reply extensions/replicate/package.json:20]: @cbh123 Do you think this new feature should be a separate command? So users would have one command for viewing past predictions and another to create new ones?

Comment (cbh123) [reply extensions/replicate/package.json]: Sure! Just pushed a commit to keep the command consistent. I think having one command is good for now so we can get this merged in!

**Comment by zeke** [comment]
Excited to see that land! 🙏🏼

**Review by pernielsentikaer — APPROVED** [review]
Hello 👋

I gave it a quick spin, works as expected, and approved 🔥

**Pull request merged**